server/aws-lsp-identity/src/awsError.test.ts

```diff
@@ -0,0 +1,55 @@
+import { AwsErrorCodes } from '@aws/language-server-runtimes/server-interface'
+import { AwsError, tryAsync } from './awsError'
+import { expect, use } from 'chai'
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+use(require('chai-as-promised'))
+
+describe('AwsError', () => {
+    it('Wraps general error in AwsError with given awsErrorCode', () => {
```

Comment:
Should we add a test case to check `AwsError` is instanceof `Error` with `extra properties`?